lib/dataset-tools/parsers.ts

```diff
@@ -266,7 +275,7 @@ class DialogueParser extends Stream.Transform {
             if (/^(R)?(P)?(C)?(S)?(E)?$/.test(this._id))
                 this._id = '_' + this._id;
         }
-        if (!line || line.startsWith('#')) {
+        if (!line) {
```

Comment:
So, the first comment line is parsed both as the ID of the dialogue and the comment for the first turn?
Also, how do comment work if they're not in the middle of a turn, they get assigned to whichever turn follows them?